tools/e2e-commons/pages/page-actions.js

```diff
@@ -29,7 +29,7 @@ export default class PageActions {
 			throw new Error( 'Cannot navigate! Page URL is not set' );
 		}
 		logger.action( `Navigating to ${ url }` );
-		await this.page.goto( url, options );
+		return await this.page.goto( url, options );
```

Comment:
Is this intentional?

Reply:
Yes.

We are relying on reading data from the response (the http status and the body), and the respons gets returned from `this.page.goto`, but was previously just tossed here.

We hoped not to break any other tests with it and according to the checks on this PR we didn't.

Do you think it will be an issue?

Reply:
Oh, right. I didn't realize that `page.goto` actually returns a meaningful/useful result. All good then :)